deps/libff/LICENSE.libff

```diff
@@ -0,0 +1,18 @@
+libff is licensed under the ISC license. The license terms are fully
```

Comment:
libff has been removed since this PR was opened. This addition is no longer necessary.